tools/test/travis-ci/scancode_evaluate_test.py

```diff
@@ -81,6 +95,7 @@ def test_missing_license_permissive_license_and_spdx(self, create_scanned_files)
             test3.h: Missing `Permissive` license text and `spdx` in match.identifier and not in file tested by ScanCode (error count += 1)
             test4.h: Missing `Permissive` license text and `spdx` in match.identifier but found in file tested by ScanCode (error count += 1)
             test5.h: Missing `spdx` in match.identifier but found in file tested by ScanCode. (error count += 0)
+            test6.h: Matching 'unknown-spdx' in match.identifier and Permissive Binary License in header (error count += 1)
             @inputs scancode_test/scancode_test_2.json
             @output 3
```

Comment:
```suggestion
            @output 4
```
You added an extra error so how does the test still pass? Shouldn't the expected value also be updated from 3 to 4?

Reply:
The docstring isn't right, hadn't updated it after our changes. I'll change it now.